### ISSUE TITLE
Schematic transparency

### ISSUE THREAD

**Issue by alex-Symbroson** [maruohon/litematica#253]
When displaying a schematic in an area with complete different/untouched terrain or on top of already existing buildings I cant navigate/see what I have to remove.

Please support an transparency setting for schematic placements.

**Comment by maruohon** [comment]
Translucent schematic rendering is already possible, for that in Visuals enable the `renderBlocksAsTranslucent` option and set the alpha value in `ghostBlockAlpha`.

However if you mean just for clearing out areas before starting to build, then I would recommend instead that you disable both the schematic block rendering and the missing block overlay types, that way you only get the red or purple overlay around blocks that you have to mine away. That's by far the best option for clearing out areas before starting to build. So in Visuals disable `enableSchematicBlocksRendering` and `schematicOverlayTypeMissing`.

**Comment by sweetbubbles** [comment]
I've tried flipping the renderBlocksAsTranslucent option on and off and the blocks are always shown as non-translucent, would you know why? I'm on 1.17 with the 1.17 litematica and fabric

**Comment by maruohon** [comment]
The translucent rendering simply does not work in the current 1.17 version, due to the vanilla rendering changes in 1.17 making the old approach now non-functional.

I'll look into fixing it at some point, but I'm not actively working on the 1.17 versions currently, as I want to rather work on the mod code rewrites and clean-up that is happening in 1.12.2. There are lots of various other issues that will benefit from those rewrites or bugs that are only properly fixable by rewriting some things.

**Comment by LyricArt** [comment]
.... Did you find a fix???
Lol I keep falling through because I just can't seem to make the blocks more transparent.

**Comment by SkySpnc** [comment]
i think this bug was never fixed honestly. its still opaque even with alpha transparency set. still present on 1.21.4 and has been on prior versions in my modpack too

<img width="3840" height="2091" alt="Image" src="https://github.com/user-attachments/assets/ae77a5f1-78cf-4075-ac3f-7d2d9545a192" />

**Comment by maruohon** [comment]
@SkySpnc It should be mostly working in most of the recent versions, but it might just not work at all (or it could look horribly wrong) if you use shaders, depending on the shader pack.

**Comment by SkySpnc** [comment]
yeah i completely forgot about shaders until my partner mentioned it to me honestly. disabling them made it transparent again lol

**Comment by nicolas-goyon** [comment]
Could be set a "solved" by disabling shader, however a suggestion would add a disclamer on both the ghost alpha and enable translucent option to say something like : "if you use shader, you may have to disable it for this to work"
